languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/pt-PT/grammar.xml

```diff
@@ -1423,16 +1423,27 @@
         </antipattern>
         <!-- MARCOAGPINTO 2022-08-03 (Checked/Enhanced) (25-JUL-2022+) *END* -->
 
-        <!-- MARCOAGPINTO 2022-11-25 (Checked/Enhanced) (25-JUL-2022+) *START* -->
         <antipattern>
-          <token postag='SENT_START|_PUNCT' postag_regexp='yes'/>
+          <token postag='SENT_START|_PUNCT|I|SPS00:DA.+' postag_regexp='yes'>
+              <exception regexp='yes'>como|já|mas|também</exception> <!-- Used specific words because POSes remove valid entries -->
+          </token>
           <token postag='DP1.+' postag_regexp='yes'/>
-          <token regexp='yes'>car[ao]s?|estimad[ao]s?|excelentes?|excelentíssim[ao]s?|querid[ao]s?</token>
+          <token regexp='yes'>adoráv(el|eis)|amantes?|amig[ao]s?|car[ao]s?|clientes?|colaborador(es)?|colegas?|companheir[ao]s?|empregad[ao]s?|estimad[ao]s?|excelentes?|excelentíssim[ao]s?|filh[ao]s?|funcionári[ao]s?|inimig[ao]s?|irmã(s|o|os)?|mães?|médic[ao]s?|pais?|prim[ao]s?|professor(res)?|querid[ao]s?</token>
```

Comment:
**[nitpick 🥜]** to avoid this type of thing, why not just a list of words with a POS tag filter for nouns/adjectives, and `inflected="yes"`?